app/helpers.php

```diff
@@ -190,3 +192,63 @@ function recursive_array_search_insert($value, &$node, $insert = '') {
         }
     }
 }
+
+if (!function_exists('getEducationalLevel')) {
+    function getEducationalLevel($grade)
+    {
+        // Range: -1 to 13 
+        // Note: Pre-K = -1, Kindergarten = 0, Adult = 13 
+        $grades = [
+            'Kindergarten-Grade 2 (Ages 5-7)' => [0, 2],
+            'Grades 3-5 (Ages 8-10)' => [3, 5],
+            'Grades 6-8 (Ages 11-13)' => [6, 8],
+            'Grades 9-10 (Ages 14-16)' => [9, 10],
+            'Grades 11-12 (Ages 16-18)' => [11, 12],
+            'College & Beyond' => [13],
+            'Professional Development' => [13],
+            'Special Education' => [13]
+        ];
+        return array_key_exists($grade, $grades) ? $grades[$grade] : [0, 2]; 
+    }
+}
+
+if (!function_exists('getFrontURL')) {
+    function getFrontURL() {
```

Comment:
Opening bracket for function on same line unlike for other function where its moved on next line.

Reply:
Formatting fixed.